web/src/hooks/theme.js

```diff
@@ -21,7 +21,8 @@ export function useCo2ColorScale() {
   const theme = useTheme();
 
   return useMemo(
-    () => scaleLinear().domain(theme.co2Scale.steps).range(theme.co2Scale.colors).unknown('gray').clamp(true),
```

Comment:
The `gray` unknown color actually doesn't represent the gray we use on the map. This is now fixed. It enables to simplify the code in `ZoneMap` by simply calling the colorscale and not bothering to check if the input value is null.